ecs-agent/tmds/handlers/fault/v1/types/types.go

```diff
@@ -13,8 +13,45 @@
 
 package types
 
+import "fmt"
+
 const (
-	BlackHolePortFaultType = "network-blackhole-port"
-	LatencyFaultType       = "network-latency"
-	PacketLossFaultType    = "network-packet-loss"
+	BlackHolePortFaultType    = "network-blackhole-port"
+	LatencyFaultType          = "network-latency"
+	PacketLossFaultType       = "network-packet-loss"
+	missingRequiredFieldError = "required parameter %s is missing"
 )
+
+type NetworkFaultRequestInterface interface {
```

Comment:
nit: Do we need to include "Interface" in the name?

Reply:
No we don't. I wanted to make it explicit but can rename it to just `NetworkFaultRequest`